csharp/ql/test/query-tests/Nullness/E.cs

```diff
@@ -385,6 +385,18 @@ static bool Ex37(E e1, E e2)
             return true;
         return e1.Long == e2.Long; // GOOD (false positive)
     }
+
+    int Ex38(int? i)
+    {
+        i ??= 0;
+        return i.Value; // GOOD
```

Comment:
I'm not sure if the check covers this case or not. Maybe we should add a test for it:
```
int? i = 1;
i ??= null;
return i.Value; // BAD
```

Reply:
Good idea; and that actually revealed another FP which I will fix on this PR as well.